masu/processor/aws/aws_report_processor.py

```diff
@@ -199,6 +200,38 @@ def process(self):
         LOG.info('Completed report processing for file: %s and schema: %s',
                  self._report_name, self._schema_name)
 
+    def remove_temp_cur_files(self, report_path):
```

Comment:
Is there some reason we can't use `shutil.rmtree()` to just nuke the entire directory?